src/main/java/basic/Exercise03.java

```diff
@@ -0,0 +1,16 @@
+package basic;
+
+public class Exercise03 {
+    public static String primeFactorization(int n) {
+        StringBuilder result = new StringBuilder();
```

Comment:
```suggestion
        final StringBuilder result = new StringBuilder();
```

Do the same for other places